### PR TITLE
Enable ast-check by default

### PR DESCRIPTION
**Pull request by Techatrix**
Suggested by @kristoff-it

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -103,7 +103,7 @@ The following options are currently available.
 | --- | --- | --- | --- |
 | `enable_snippets` | `bool` | `false` | Enables snippet completions when the client also supports them. |
 | `enable_unused_variable_warnings` | `bool` | `false`| Enables warnings for local variables that aren't used. |
-| `enable_import_embedfile_argument_completions` | `bool` | `false` | Whether to enable import/embedFile argument completions |
```
Comment (SuperAuguste) [inline README.md:106]: You swapped out the wrong option in the README! :P

**Review by SuperAuguste — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Techatrix) [reply README.md:106]: sorry ( ͡° ʖ̯ ͡°)

**Review by SuperAuguste — APPROVED** [review]

**Comment by nektro** [comment]
is the running of `zig fmt` irrespective of these diagnostics?

**Comment by SuperAuguste** [comment]
No; `zig fmt` and `zig ast-check` are completely independent! :)

**Pull request merged**